flake.nix

```diff
@@ -8,24 +8,30 @@
       url = "github:edolstra/flake-compat";
       flake = false;
     };
+    solidity = {
+      url = "github:ethereum/solidity/1c8745c54a239d20b6fb0f79a8bd2628d779b27e";
```

Comment:
not a big deal but the commit is already in flake.lock, will be easier to upgrade

Reply:
I pinned the commit becuase I didn't want this to be bumped when we run `nix flake update`, we need to manually review each update to the solidity repo as it may include new tests that need to be ignored.